TGM/src/main/java/network/warzone/tgm/command/PunishCommands.java

```diff
@@ -229,6 +229,30 @@ public static void staffchat(CommandContext cmd, CommandSender sender) {
         Bukkit.getConsoleSender().sendMessage(result);
     }
 
+    @Command(aliases = {"chat"}, desc = "Control chat settings", min = 1, usage = "(mute|clear)")
+    @CommandPermissions({"tgm.chat.control"})
+    public static void chat(CommandContext cmd, CommandSender sender) {
+        String action = cmd.getString(0);
+        if (action.equalsIgnoreCase("mute")) {
```

Comment:
Add a third argument so we can set this to true or false

Reply:
The mute command will unmute the chat if it's already muted.